axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
Tricky to do - anything you write needs to take into account the fact that the dictionary will be overridden by all the inherited classes and, unless they have all dimensions specified, you will hit missing key errors.